modules/openapi-generator/src/main/resources/Java/pojo.mustache

```diff
@@ -58,7 +44,15 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#parcela
   public static final String SERIALIZED_NAME_{{nameInSnakeCase}} = "{{baseName}}";
   @SerializedName(SERIALIZED_NAME_{{nameInSnakeCase}})
   {{/gson}}
+  {{#jackson}}
+{{> jackson_annotations}}
+  {{/jackson}}
+  {{#vendorExtensions.isJacksonOptionalNullable}}
+  private JsonNullable<{{{datatypeWithEnum}}}> {{name}} = JsonNullable.{{#defaultValue}}of({{{.}}}){{/defaultValue}}{{^defaultValue}}undefined(){{/defaultValue}};
```

Comment:
Is there a need for a vendor extension ? I think it can be wrapped with JsonNullable as soon as the field is "nullable".

Reply:
This was my line of thinking with the vendor extension:

I needed to have a bunch of "if and else" conditions in the pojo template to determine if the context have `jackson`, `!required` and `nullable` set at the same time. I could theoretically drop the `!required` as condition (even though I like having the simplicity of going with `JsonInclude.Include.ALWAYS` for required fields), but I can't drop the `jackson` as condition.
If I didn't have this as one variable in the mustache rendering context, it'd be hard doing the "else" part. Additionally, since `jackson` is Java specific, I decided to use a "vendor" variable for this, as the other uses of vendor extensions in other codegens seemed to be similar to what I'm doing here.

I'm of course ok with improving this if you have a better idea.